### ISSUE TITLE
Line length & numbers varies when \jlreqkanjiskip changed and \warichu used

### ISSUE THREAD

**Issue by Kuroroholumu** [abenori/jlreq#107]
I'll try to express the issue with Japanese language:

古籍刻本のように全ての漢字が完璧な整列になるように`\renewcommand{\jlreqkanjiskip}{.0\zw plus.0\zw minus .0\zw}`を入りました。そして`\warichu`をするとき、縦書きの行の長さが一文字足りなかったり多くになった入り、行数が一行なくなったり、うまく整列できない状況は発生しています。`\jlreqkanjiskip`の設定を元に戻すなら整列できないが組版の動作は正常に戻ります。

一文字多くの例：
![L%}I8~89NH9Z)(MB_WQL12H](https://user-images.githubusercontent.com/31466085/158027689-1e786e31-69d5-43c2-9abb-29d9de96ee70.png)

**Comment by Kuroroholumu** [comment]
> Can you given the source file of the example?
[sample.zip](https://github.com/abenori/jlreq/files/8262119/sample.zip)
Thank you for your reply.
Here is a simple example but not exactly from what I am doing right now.
The whole project is a littile complicate, including some modified vf/tfm system and special fonts, and recently I'm reconstructing it. 😓

**Comment by abenori** [comment]
I saw the issue. Although I'm not 100% sure, I guess it is because of the mechanism of `\warichu`. `\warichu` tries to find a good place for breaking, but this does not work well sometimes, and I think it is the case. (Maybe I should improve this.)
Why don't you use `\warichu*`. Users can specify the breaking points with this and TeX will not try to find them.

**Comment by Kuroroholumu** [comment]
My idea is that two different typestting styles could be applied to a same manuscript. So if using `\warichu*` I need break the note part by specific marks in the manuscript. That should cause problems when using another typestting style.

Recently I found an old package named [gezhu](https://github.com/yang-le/gezhu) and it works well so far.  And I also uploaded the my typesetting repos to Github. 

Anyway thank you for your help and amazing jlreq document class. Since this issue is really not much about the jlreq's original purpose, I think we could just close this issue.
